### PR TITLE
Us 03 search product by category desktop

### PR DESCRIPTION
**Pull request by Ludovicscelles**
US-03_search-product-by-category_desktop

### CODE REVIEW

**Review by NicosMQ — APPROVED** [review]

File: client/package.json [hunk 1]
```diff
@@ -14,8 +14,12 @@
     "prop-types": "^15.8.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "react-router-dom": "^6.23.0",
+
+    "react-router-dom": "^6.23.1"
+=======
```
Comment (Alexandre78R) [inline client/package.json:19]: Delete total conflit  ======

File: server/app/routers/api/items/router.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const {
   add,
   edit,
   destroy,
+  // getUserByItem,
```
Comment (Alexandre78R) [inline server/app/routers/api/items/router.js]: delete code dead

**Review by Alexandre78R — CHANGES_REQUESTED** [review]
Delete code

**Review by Alexandre78R — APPROVED** [review]
Good

**Pull request merged**